sonaranalyzer-dotnet/its/expected/ManuallyAddedNoncompliantIssues/IntentionalFindings--S2755.json

```diff
@@ -1,30 +0,0 @@
-{
-"issues":  [
-{
-"id":  "S2755",
-"message":  null,
-"location":  {
-"uri":  "sources\ManuallyAddedNoncompliantIssues\IntentionalFindings\S4635.cs",
```

Comment:
this was raised when there was a dummy implementation that was raising on everything... sorry for the noise